main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+        )
+        fl.addRow('Center V:', center_v)
+        center_u = QDoubleSpinBox()
+        center_u.setRange(0.0, 10000.0)
+        center_u.setDecimals(1)
+        center_u.setValue(p.get('center_u', 0.0))
+        center_u.valueChanged.connect(
+            lambda v, i=idx: self._on_body_field(i, 'center_u', v)
+        )
+        fl.addRow('Center U:', center_u)
+        # Keep references so drag updates can sync the UI
+        setattr(w, 'center_v_spin', center_v)
```

Ruff (0.14.5)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)